src/thundra-wrapper.js

```diff
@@ -100,19 +103,75 @@ class ThundraWrapper {
     async report(error, result, callback) {
         if (!this.reported) {
             this.reported = true;
-            const afterInvocationData = {
+
+            let afterInvocationData = {
                 error: error,
                 response: result
             };
+
+            const skipParseResponse = process.
+                env.thundra_lambda_skip_parse_response === 'true' ? true : false;
+
+            if (skipParseResponse || this.isErrorResponse(result)) {
+                afterInvocationData = {
+                    error: new Error("Lambda returned with error response."),
+                    response: null
```

Comment:
Assign `result` to `response` event though there is error. So user can see which status code was returned

Reply:
👍